src/x86/sse41.rs

```diff
@@ -627,6 +647,13 @@ extern "C" {
     fn roundsd(a: f64x2, b: f64x2, rounding: i32) -> f64x2;
     #[link_name = "llvm.x86.sse41.round.ss"]
     fn roundss(a: f32x4, b: f32x4, rounding: i32) -> f32x4;
+    #[link_name = "llvm.x86.sse41.phminposuw"]
+    fn phminposuw(a: u16x8) -> u16x8;
+    #[link_name = "llvm.x86.sse41.pmuldq"]
+    fn pmuldq(a: i32x4, b: i32x4) -> i64x2;
+    // pmulld is not supported by llvm so far
+    // #[link_name = "llvm.x86.sse41.pmulld"]
+    // fn pmulld(a: i32x4, b: i32x4) -> i32x4;
```

Comment:
This is intentionally not supported by LLVM because we're supposed to implement it using their portable vector API. As #40 says, do what Clang does: https://github.com/llvm-mirror/clang/blob/a38ba9770a70056f9fdd6c71f819e5db45a105e4/lib/Headers/smmintrin.h#L546-L564